components/Events.vue

```diff
@@ -46,22 +50,63 @@ export default {
   padding-top: 17%;
 }
 
+#scaledSection h3.title {
```

Comment:
`#scaledSection` seems like an odd selector to use. Is this meant for the entire page? Or just the events section? If its the latter maybe we could use `.events` or something instead?

Reply:
I just used a random #id selector to specificity-override mixins.sass which forces a margin-bottom of 1.5em

Reply:
Doesn't really matter anymore but I think it should not be using #scaledSection